extensions/commands/art/cmd_property.py

```diff
@@ -37,10 +37,12 @@ def _get_path_from_ref(ref):
         recipe_ref = RecipeReference.loads(ref)
         package_ref = None
 
-    rrev_path = f"/_/{recipe_ref.revision}" if recipe_ref.revision else ""
+    user = recipe_ref.user if recipe_ref.user else "_"
+    channel = recipe_ref.channel if recipe_ref.channel else "_"
+    rrev_path = f"/{channel}/{recipe_ref.revision}" if recipe_ref.revision else ""
```

Comment:
Better to leave rrev_path mostly as it is and use directly the channel value in the final path formatting
```suggestion
    rrev_path = f"/{recipe_ref.revision}" if recipe_ref.revision else ""
```